### PR TITLE
Add mxnet android template

### PR DESCRIPTION
**Pull request by stu1130**
MXNet Android template

### CODE REVIEW

File: mxnet-android/.gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
```
Comment (frankfliu) [inline mxnet-android/.gitignore:4]: .idea/

File: mxnet-android/.gitignore [hunk 2]
```diff
@@ -0,0 +1,14 @@
+*.iml
+.gradle
+/local.properties
```
Comment (frankfliu) [inline mxnet-android/.gitignore:3]:
```suggestion
local.properties
```

File: mxnet-android/.gitignore [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
```
Comment (frankfliu) [inline mxnet-android/.gitignore:11]:
```suggestion
build
```
```suggestion
/build
```

File: mxnet-android/.gitignore [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+/captures
```
Comment (frankfliu) [inline mxnet-android/.gitignore:12]: What's this?

File: mxnet-android/README.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+Depending on Android you would like to test, you just need to put the corresponding `libmxnet.so` under the *src/main/jniLibs/{ABI}*.
+For example, if you want to run on `armeabi-v7a`, put the right `libmxnet.so` under *src/main/jniLibs/armeabi-v7a*.
+
+Then you're all set. You can run this project and should be able to see the classification result right in the middle.
+
+Here are extra setting I did when I created the project.
+
+1. In `build.gradle`, I included 4 packages `ai.djl:api`, `ai.djl.android:core`, `ai.djl.mxnet:mxnet-engine` and `ai.djl.mxnet:mxnet-model-zoo`
+to use DJL API. Because DJL MXNet interact with MXNet engine via JNA, one more dependency called `net.java.dev.jna:jna` is required as well.
+Originally, JNA was packaged along with DJL api. We have to exclude duplicate JNA dependencies by using `exclude group: "net.java.dev.jna", module: "jna"`
```
Comment (frankfliu) [inline mxnet-android/README.md:185]:
```suggestion
Originally, JNA was packaged along with DJL api. We have to exclude duplicate JNA dependencies by using `exclude group: "net.java.dev.jna", module: "jna"`

DJL api depends on JNA by default. For Android project, we should use `net.java.dev.jna:jna:5.3.0@aar` instead. We need to exclude duplicate JNA dependencies by using `exclude group: "net.java.dev.jna", module: "jna"`
```

**Review by frankfliu — APPROVED** [review]

File: mxnet-android/README.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
+# DJL MXNet Android Template
+
+### Introduction
+This template includes 1. how you build the mxnet native library(*libmxnet.so*) for android
+2. how you use MXNet native library(*libmxnet.so*) in Android Studio with DJL to run a simple image classification.
+DJL team also prepare an unofficial prebuilt `libmxnet.so` on 4 common Android ABI.
```
Comment (goswamig) [inline mxnet-android/README.md:6]: Does MXNet not release android compatible binary ?

**Comment by lanking520** [comment]
Can we make it as a part of Android project?

File: mxnet-android/README.md (hunk 6 above)
Comment (stu1130) [reply mxnet-android/README.md:6]: no

File: mxnet-android/.gitignore (hunk 4 above)
Comment (stu1130) [reply mxnet-android/.gitignore]: It is generated by android studio

**Comment by stu1130** [comment]
> Can we make it as a part of Android project?

done

**Review by lanking520 — APPROVED** [review]

**Pull request merged**